### PR TITLE
[Bug]fix path split exception in win10,not check file existed

### PR DESCRIPTION
**Pull request by laglangyue**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by laglangyue** [comment]
close #2712

**Comment by laglangyue** [comment]
I has test in the Environment : Mycomputer is WIN10,and docker in virtual box(Centos7 docker),
 and I will test on 9.10 afternoon in ubuntu22.04，because my computer is in company.
Besides I can't test it in Mac

File: seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/AbstractContainer.java [hunk 1]
```diff
@@ -42,15 +43,14 @@ public abstract class AbstractContainer {
     protected final String startModuleFullPath;
 
     public AbstractContainer() {
-        String[] modules = getStartModulePath().split(File.separator);
-        this.startModuleName = modules[modules.length - 1];
-        this.startModuleFullPath = PROJECT_ROOT_PATH + File.separator +
-            START_ROOT_MODULE_NAME + File.separator + getStartModulePath();
+        this.startModuleName = getStartModuleName();
+        this.startModuleFullPath = PROJECT_ROOT_PATH + File.separator + START_ROOT_MODULE_NAME + File.separator + getStartModuleName();
```
Comment (TyrantLucifer) [inline seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/AbstractContainer.java]: change `getStartModuleName` to `this.startModuleName`

**Comment by laglangyue** [comment]
I has test it in Ubuntu22.04, docker is local,it's ok.

File: seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/AbstractContainer.java (hunk 1 above)
Comment (laglangyue) [reply seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/AbstractContainer.java]: done

**Review by ashulin — APPROVED** [review]
LGTM

**Review by TyrantLucifer — APPROVED** [review]
LGTM.

**Pull request merged**